worlds/dlcquest/Items.py

```diff
@@ -74,7 +75,7 @@ def load_item_csv():
 
 
 def initialize_item_table():
-    item_table.update({item.name: item for item in all_items})
+    item_table.update({item.name: item for item in all_items if datetime.datetime.today().month == 4 or Group.Piece not in item.groups})
```

Comment:
This line is longer than 120 characters

Reply:
As far as I know we are not held to a specific standard on that front

Reply:
https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/style.md#python-code
> 120 characters per line.

Reply:
Resolved because I agree for this specific instance, but here's an interesting conversation about worlds not being obligated to follow these conventions specifically https://discord.com/channels/731205301247803413/1214608557077700720/1264982709986525214